### PR TITLE
testing/py3-hbmqtt: new aport

### PR DESCRIPTION
**Pull request by fabaff**
MQTT client/broker using Python asynchronous I/O
http://hbmqtt.readthedocs.io/

Required by #506.

### CODE REVIEW

File: testing/py3-websockets/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+url="https://websockets.readthedocs.io"
+arch="noarch"
+license="BSD"
+makedepends="python3-dev"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python3 setup.py build || return
```
Comment (fabled) [inline testing/py3-websockets/APKBUILD]: missing return error code

File: testing/py3-hbmqtt/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+arch="noarch"
+license="MIT"
+depends="py3-passlib py3-websockets py3-transitions py3-docopt py3-yaml"
+makedepends="python3-dev"
+source="$_pkgname-$pkgver.tar.gz::https://github.com/beerfactory/hbmqtt/archive/$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python3 setup.py build || return
```
Comment (fabled) [inline testing/py3-hbmqtt/APKBUILD]: missing return error code

File: testing/py3-hbmqtt/APKBUILD [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py3-hbmqtt
 _pkgname=hbmqtt
-pkgver=0.7.3
+pkgver=0.8
```
Comment (fabled) [inline testing/py3-hbmqtt/APKBUILD]: Please flatting this commit with the previous one.

File: testing/py3-websockets/APKBUILD (hunk 1 above)
Comment (fabaff) [inline testing/py3-websockets/APKBUILD]: Thanks

File: testing/py3-hbmqtt/APKBUILD (hunk 3 above)
Comment (fabaff) [inline testing/py3-hbmqtt/APKBUILD]: Squashed, thanks.

**Pull request closed without merging**